src/app/containers/Copyright/index.test.jsx

```diff
@@ -13,6 +13,7 @@ const newsServiceContextStub = {
   imageCopyrightOffscreenText: 'Image source, ',
```

Comment:
In addition to Drew's comment -  adding `dir: 'ltr'` in the stubs for the LTR unit test cases.